src/test/java/com/snack/news/service/SlackBotServiceTest.java

```diff
@@ -0,0 +1,20 @@
+package com.snack.news.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+
+@ExtendWith(MockitoExtension.class)
+class SlackBotServiceTest {
+
+	@Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5471424)